src/components/App/index.js

```diff
@@ -313,7 +313,7 @@ class App extends React.Component {
         });
         this.mapView.drawDonationMarker(state);
       }
-      this.mapView.map.setZoom(7);
+      this.mapView.map.setZoom(3);
```

Comment:
Can you add a little comment to explain why are we doing this even if 3 is the default zoom?